ajax/libs/jodit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jodit",
-  "version": "2.5.62",
+  "version": "3.0.8",
```

Comment:
Let's reverse to `2.5.62`. 
After this PR merge, the auto-update config will get the files from new versions, and also update this field.